satpy/readers/msi_safe.py

```diff
@@ -114,6 +114,9 @@ def __init__(self, filename, filename_info, filetype_info):
         self.tile = filename_info['gtile_number']
         self.platform_name = PLATFORMS[filename_info['fmission_id']]
 
+        import geotiepoints  # noqa
+        import bottleneck  # noqa
```

Comment:
Do we normally provide a custom ImportError for readers when they need these so check_satpy has pretty messages?